src/test/kotlin/io/bazel/kotlin/builder/tasks/jvm/KotlinKSPMetaFilesCopyTest.kt

```diff
@@ -0,0 +1,30 @@
+package io.bazel.kotlin.builder.tasks.jvm
+
+import io.bazel.kotlin.KotlinAssertionTestCase
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.junit.runners.JUnit4
+
+@RunWith(JUnit4::class)
+class KotlinKSPMetaFilesCopyTest: KotlinAssertionTestCase("src/test/data/jvm/ksp") {
```

Comment:
I don't think the BUILD file for this got pushed.

Reply:
Pushed.